<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="CPlayWidget" name="wdg1"/>
   </item>
   <item row="0" column="1">
    <widget class="CPlayWidget" name="wdg2"/>
   </item>
   <item row="0" column="2">
    <widget class="CPlayWidget" name="wdg3"/>
   </item>
   <item row="1" column="0">
    <widget class="CPlayWidget" name="wdg4"/>
   </item>
   <item row="1" column="1">
    <widget class="CPlayWidget" name="wdg5"/>
   </item>
   <item row="1" column="2">
    <widget class="CPlayWidget" name="wdg6"/>
   </item>
   <item row="2" column="0">
    <widget class="CPlayWidget" name="wdg7"/>
   </item>
   <item row="2" column="1">
    <widget class="CPlayWidget" name="wdg8"/>
   </item>
   <item row="2" column="2">
    <widget class="CPlayWidget" name="wdg9"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CPlayWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>gplaywidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
